### ISSUE TITLE
Allow for custom media player

### ISSUE THREAD

**Issue by drewauff** [mafintosh/peerflix#381]
I haven't been able to figure it out.

Is it possible to stream to a custom media player that is not listed in the switches?

If not I think it would be a great addition.

**Comment by stl3** [comment]
If you don't pass a player argument, by default it will just host it at http://your.ip:port/
You could also paste the full name that it shows with the file, just append it to the http above, for example http://your.ip:port/whatever.file.is.streaming.mkv
From there, you can either try to paste the url into another player, for example it worked for me in Daum Potplayer. If you want to have more silly fun, you can open it in vlc, and then restream it to somewhere else, like a chromecast or tv, although you can also get the tv to reach it as a dlna if I'm not mistaken. What player were you thinking of using?

**Comment by drewauff** [comment]
ahh I see..Oh I have a fork of mpv which was converted to a MacOS app, I have got other "scripts/programs" to work with it by point them straight at the executable rather that the .app which wont work.

Getting it working is simple enough, just have to find in the lin in the code which the player(s) is define and change it.

**Comment by stl3** [comment]
Yes there should be a few ways you could get around it like what you mentioned. Let me know if you managed to get it going. I don't have macos to test with but I like to see workarounds.
